### DISCUSSION TITLE
Slack alerts redirecting to localhost - Seq in linux docker

### DISCUSSION THREAD

**Discussion by mslupevents** [datalust/seq-tickets#2276] [Help]
I Set up Seq server in a docker container running on Amazon Linux 2023
alongside I have a caddy container for TLS termination 
both containers are in the same docker bridge network

Everything works well apart from the Slack Notifier which is redirecting to `localhost:80`
SEQ_API_CANONICALURI is already set when running docker

What am I missing?

**Comment by liammclennan** [comment]
Hi @mslupevents 

Please send a diagnostic report (Settings > Diagnostics) to support@datalust.co and we will have a look.
